### PR TITLE
Ignore the file '/etc/board-release'

### PR DESCRIPTION
**Pull request by pedrolamas**
Adds `board-release` to ignored patterns, which is used by some Armbian based releases, such as the one used with the [BigTreeTech CB1](https://github.com/bigtreetech/CB1-Kernel).

Full text search here:
https://github.com/bigtreetech/CB1-Kernel/search?q=board-release

### CODE REVIEW

**Comment by HorlogeSkynet** [comment]
Hi @pedrolamas ! Many thanks for your PR, one comment though : could you provide a "testing" rootfs under `tests/resources/distros/` too ? 🙂

**Comment by pedrolamas** [comment]
Thanks @HorlogeSkynet, I've now added a test for this specific case.

**Comment by HorlogeSkynet** [comment]
Thanks ! We'll have to wait for #355 and a proper rebase/merge of this branch 😉

File: tests/resources/distros/bttcb1/etc/os-release [hunk 1]
```diff
@@ -0,0 +1,9 @@
+PRETTY_NAME="BTT-CB1 2.3.1 Bullseye"
```
[the remark this answers is not in the document]
Comment (pedrolamas) [reply tests/resources/distros/bttcb1/etc/os-release:1]: Looking at my own system, doesn't seem to be a symlink, but happy to add it like that if that still makes sense!

image

[the remark this answers is not in the document]
Comment (HorlogeSkynet) [reply tests/resources/distros/bttcb1/etc/os-release:1]: Fair enough ! Let's leave it as is then :ok_hand:

**Review by HorlogeSkynet — DISMISSED** [review]

**Review by HorlogeSkynet — APPROVED** [review]

**Pull request merged**